src/ui/map.js

```diff
@@ -37,22 +37,22 @@ define(function(require, exports, module) {
     this.configureMap = function(ev, config) {
       this.trigger('mapStarted', {});
       // if list or facets are emabled, give the map less space
-      var addition = 0;
-      if (config.facets) {
-        addition += 300;
-      }
-      if (config.list) {
-        addition += 300;
-      }
-
-      if (addition > 0) {
-        window.setTimeout(function() {
-          if (this.map) {
-            this.$node.css('left', '+=' + addition);
-            this.map.invalidateSize();
-          }
-        }.bind(this), 50);
-      }
+      // var addition = 0;
```

Comment:
@paulswartz I'm unsure how we should handle this in conjunction with a variable -width sidebar based on screen size. Maybe it's possible to adjust the "addition" via a class + media queries, which could get set based on what is enabled (list/facets)?